test/mocks/data/user_comment_data.dart

```diff
@@ -0,0 +1,35 @@
+import "dart:math";
+
+import "package:proxima/models/database/comment/comment_id_firestore.dart";
+import "package:proxima/models/database/post/post_id_firestore.dart";
+import "package:proxima/models/database/user_comment/user_comment_data.dart";
+
+class UserCommentDataGenerator {
+  final Random _random;
+
+  UserCommentDataGenerator({int seed = 0}) : _random = Random(seed);
+
+  List<UserCommentData> generateUserCommentData(int count) {
```

Comment:
Same comment here.

Reply:
I think what you did is completely fine, we have in multiple places two functions for mocks, one that generate a single element, other that generate a list of elements. I just think those two functions should have a more similar name. For instance, `generateUserCommentData` and `generateUserCommentDatas`.

Reply:
Oh ok I see.
That is a good idea indeed.
Done in 0b38809867c27818b007d46ce63717853e0f2bd4.